examples/transforms/align.rs

```diff
@@ -2,6 +2,7 @@
 
 use bevy::color::palettes::basic::{GRAY, RED, WHITE};
 use bevy::input::mouse::{AccumulatedMouseMotion, MouseButtonInput};
+use bevy::math::StableInterpolate;
```

Comment:
I suppose it might be useful to have a doc comment somewhere explaining the point of this import, for new users.